ieee.csl

```diff
@@ -201,7 +212,7 @@
               <group delimiter=" " prefix="(" suffix=")">
                 <text term="accessed"/>
                 <date variable="accessed">
-                  <date-part name="month" form="short" strip-periods="false"/>
+                  <date-part name="month" form="short" strip-periods="false" suffix="."/>
```

Comment:
why are you adding the period suffix here? The month abbreviation contains a period.

Reply:
I'll do some more testing/investigation and get back to you. It wasn't included (hence why I added it). May be a localization thing?

Reply:
Ok, I think this may have been because I was testing with citation that was released in May...

It does seem to work fine (although I did update Zotero recently, so it's possible there was an issue but it's been resolved. Either way I'll delete this.